framework/helpers/BaseStringHelper.php

```diff
@@ -45,6 +45,23 @@ public static function byteSubstr($string, $start, $length = null)
     {
         return mb_substr($string, $start, $length === null ? mb_strlen($string, '8bit') : $length, '8bit');
     }
+    
+    /**
+     * Returns given string with first character uppercased
+     * @param string $string
+     * @param string $encoding
+     * @return string
+     */
+    public static function ucfirst($string, $encoding = null)
+    {
+        if (!$encoding) {
+            $encoding = \mb_internal_encoding();
+        }
+        $strlen = mb_strlen($string, $encoding);
+        $firstChar = mb_substr($string, 0, 1, $encoding);
+        $then = mb_substr($string, 1, $strlen - 1, $encoding);
+        return mb_strtoupper($firstChar, $encoding) . $then;
```

Comment:
There are simpler options, such as using [mb_convert_case](http://ru.stackoverflow.com/questions/105908/%D0%9F%D0%BE%D1%87%D0%B5%D0%BC%D1%83-ucfirst-%D0%BD%D0%B5-%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%B0%D0%B5%D1%82-%D0%B4%D0%BB%D1%8F-%D1%80%D1%83%D1%81%D1%81%D0%BA%D0%B8%D1%85-%D1%81%D0%B8%D0%BC%D0%B2%D0%BE%D0%BB%D0%BE%D0%B2). Have you considered using them?